fabric-chaincode-shim/src/test/java/org/hyperledger/fabric/shim/impl/InnvocationTaskManagerTest.java

```diff
@@ -112,7 +112,7 @@ void onChaincodeMessage() throws IOException {
         final InvocationTaskManager itm = InvocationTaskManager.getManager(chaincodeBase, chaincodeId);
 
         Assertions.assertThrows(
-                IllegalArgumentException.class, () -> itm.onChaincodeMessage(null),
+                NullPointerException.class, () -> itm.onChaincodeMessage(null),
```

Comment:
The code didn't change so we need to figure out why there has been this change in behavior rather than just update the test to reflect whatever is going on

Reply:
OK, so I did look into this. The first line of that test was actually trying to print a log message that referred to the `chaincodeMessage`.  If it's null, then trying to reference it in the log message is going to throw a `NullPointerException` before you even get to the `IllegalArgumentException`. So, I don't know why that wasn't being picked up/failing in the earlier version of JUnit, but I was able to get the test case switched back to its original form simply by moving the check for whether `chaincodeMessage` is `null` to before the log message is even printed out.

This causes the test to pass, as it had in earlier versions, with only a simple change to when the log message prints.